data-prepper-plugins/s3-source/build.gradle

```diff
@@ -13,10 +13,13 @@ repositories {
 
 dependencies {
     implementation project(':data-prepper-api')
-    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation project(':data-prepper-plugins:blocking-buffer')
```

Comment:
Why did you add `blocking-buffer` here? Data Prepper API provides the `Buffer` interface.

Reply:
I added this for testing S3Source using buffer.

Reply:
For testing could this import be added as `testImplementation project(':data-prepper-plugins:blocking-buffer')`?

Reply:
@sbayer55 's suggestion is the right approach for a testing dependency.